src/renderer/components/NodeDocumentation/docsMarkdown.tsx

```diff
@@ -1,25 +1,40 @@
-import { Code, Link, Text } from '@chakra-ui/react';
+import { Code, Highlight, Link, Text } from '@chakra-ui/react';
 import ChakraUIRenderer from 'chakra-ui-markdown-renderer';
-import { memo } from 'react';
+import { PropsWithChildren, memo } from 'react';
 import { Components } from 'react-markdown';
 import { useContext } from 'use-context-selector';
 import { SchemaId } from '../../../common/common-types';
 import { BackendContext } from '../../contexts/BackendContext';
+import { NodeDocumentationContext } from '../../contexts/NodeDocumentationContext';
 import { SchemaLink } from './SchemaLink';
 
 const getDocsMarkdownComponents = (interactive: boolean): Components => {
     return {
-        p: ({ children }) => {
+        p: memo(({ children }: PropsWithChildren<unknown>) => {
+            const { useNodeDocumentationSearch } = useContext(NodeDocumentationContext);
+            const { searchTerms } = useNodeDocumentationSearch;
+
+            const stringChildren = Array.isArray(children)
+                ? (children as string[]).join('')
+                : String(children);
```

Comment:
You just destroyed any and all formatting within paragraphs. No more bold, italics, links, inline-code, anything.

![image](https://github.com/chaiNNer-org/chaiNNer/assets/20878432/e3cc02eb-b76d-4b2c-8501-21e64baf7da7)

Reply:
Ugh.... That's gonna be a pain to solve